security/4.0/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Jakarta Security 4.0 (under development)"
+date: 2023-05-30
+summary: "Release for Jakarta EE 11"
+---
+Jakarta Security defines a standard for creating secure Jakarta EE applications in modern application paradigms.
+
+The release splits focuses on evolving the API in various ways, and specifically providing APIs for the authorization theme (including interceptors and an abstraction for the permission store)
+
+### New features, enhancements or additions
+* Client-cert and Digest SECURITY #120 
+* Authentication mechanism per URL SECURIY #86 
+* User choice of authentication mechanism (login with provider X, login with provider Y, etc) 
+* Multiple authentication mechanisms (try JWT, fallback to BASIC, etc) 
+* @RolesAllowed alternative 
+* Easily adding an interceptor to a build-in CDI bean
+* Authorization modules
+
+### Removals, deprecations or backwards incompatible changes
+* Remove references to the SecurityManager
+
+### Minimum Java SE Version
+**Java SE 17 or higher**
+
+# Details
+
+* [Jakarta Security 4.0 Release Record](https://projects.eclipse.org/projects/ee4j.security/releases/4.0)
```

Comment:
Hi @arjantijms 
When I opened the release record link, I see the new features, enhancements/addition,s but I couldn't find any references to the SecurityManager removal.

Reply:
The SecurityManager is not used explicitly by Jakarta Security. This concern is delegated to Jakarta Authorization, but just to be sure, I will mention it in the Jakarta Security plan too.